notebooks/UserSearchPy3.ipynb

```diff
@@ -1360,57 +1398,62 @@
     "#         print(f\" {org}..\", end='')\n",
     "        for l in possibles:\n",
     "            full_list = []\n",
-    "            try:\n",
-    "                full_list = list(gh.search_code(query=f\"org:{org} {l}\"))\n",
-    "            except Exception as e:\n",
-    "                if isinstance(e, http.client.RemoteDisconnected):\n",
-    "                    # This is \"fun\" to run into - doesn't happen very often\n",
-    "                    # so this recovery is an educated guess (the time I\n",
-    "                    # did see it, it was after a 'resumed' message from\n",
-    "                    # the clause below)\n",
-    "                    for i in range(3):\n",
-    "                        try_login()\n",
-    "                        if gh:\n",
-    "                            # re-established connection\n",
-    "                            print(f\"re-established connection on try {i+1}\")\n",
-    "                            break\n",
+    "            assume_time_out = True\n",
+    "            while assume_time_out:\n",
+    "                try:\n",
+    "                    # 2023-05-25 can't use regex in code search, so return context for further processing\n",
+    "                    full_list = list(gh.search_code(query=f\"org:{org} {l}\", text_match=True))\n",
+    "                    assume_time_out = False\n",
```

Comment:
This is where we recover the false negatives. We won't advance to the next token until we've had a successful search with the current one. This required increasing indentation, which makes for a "huge diff" - the key lines are the ones referencing `asssume_time_out`.